test/train-sets/ref/cb_bad_one_of.stderr

```diff
@@ -0,0 +1,9 @@
+Error: Error: 'bad_type' is not a valid choice for option --cb_type
```

Comment:
Can the error say what are valid choices?

Reply:
Yea I can add that in